py-polars/polars/internals/dataframe/frame.py

```diff
@@ -1498,6 +1498,26 @@ def _repr_html_(self, **kwargs: Any) -> str:
             ).render()
         )
 
+    def item(self) -> Any:
+        """
+        Return the dataframe as a scalar.
+
+        Equivalent to ``df[0,0]``, with a check that the shape is (1,1).
+
+        Examples
+        --------
+        >>> df = pl.DataFrame({"foo": [1]})
+        >>> df.item()
+        1
```

Comment:
In the comments in this PR you gave a better example of when you would actually want to use this method. Let's include that in the examples?

Reply:
True, added that for the DataFrame. The Series one is a bit trivial, but the implementation is needed for DataFrame, see also the OP here.